site/ko/alpha/guide/migration_guide.ipynb

```diff
@@ -0,0 +1,1596 @@
+        "\n",
+        "out_a = forward([1,0])\n",
+        "print(out_a)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "colab_type": "code",
+        "id": "YmE96A_1jZTg",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "out_b = forward([0,1])\n",
+        "\n",
+        "regularizer = tf.keras.regularizers.l2(0.04)\n",
+        "reg_loss = regularizer(W)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "ycDxY9nL268-"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### `tf.layers` 기반의 모델"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "K-bIk7wL48U7"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "`tf.layers` 모듈은 변수를 정의하고 재사용하기 위해 `tf.variable_scope`에 의존하는 층 함수를 포함합니다."
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "8I_qKpT73KyM"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "#### 변환 전\n",
+        "```python\n",
+        "def model(x, training, scope='model'):\n",
+        "  with tf.variable_scope(scope, reuse=tf.AUTO_REUSE):\n",
+        "    x = tf.layers.conv2d(x, 32, 3, activation=tf.nn.relu, \n",
+        "          kernel_regularizer=tf.contrib.layers.l2_regularizer(0.04))\n",
+        "    x = tf.layers.max_pooling2d(x, (2, 2), 1)\n",
+        "    x = tf.layers.flatten(x)\n",
+        "    x = tf.layers.dropout(x, 0.1, training=training)\n",
+        "    x = tf.layers.dense(x, 64, activation=tf.nn.relu)\n",
+        "    x = tf.layers.batch_normalization(x, training=training)\n",
+        "    x = tf.layers.dense(x, 10, activation=tf.nn.softmax)\n",
+        "    return x\n",
+        "\n",
+        "train_out = model(train_data, training=True)\n",
+        "test_out = model(test_data, training=False)\n",
+        "```"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "b8_Ii7CQ3fK-"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "#### 변환 후"
+      ]
+    },
+    {
+      "metadata": {
+        "colab_type": "text",
+        "id": "BsAseSMfB9XN"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "* 층을 단순하게 쌓을 경우엔 `tf.keras.Sequential`이 적합합니다. (복잡한 모델인 경우 [맞춤형 층과 모델](keras/custom_layers_and_models.ipynb)이나 [함수형 API](keras/functional.ipynb)를 참고하세요.)\n",
+        "* 모델이 변수와 규제 손실을 관리합니다.\n",
+        "* `tf.layers`에서 `tf.keras.layers`로 바로 매핑되기 때문에 일대일로 변환됩니다.\n",
+        "\n",
+        "대부분 매개변수는 동일합니다. 다른 부분은 다음과 같습니다:\n",
+        "\n",
+        "* 모델이 실행될 때 각 층에 `training` 매개변수가 전달됩니다.\n",
+        "* 원래 `model` 함수의 첫 번째 매개변수(입력 `x`)는 사라집니다. 층 객체가 모델 구축과 모델 호출을 구분하기 때문입니다.\n",
+        "\n",
+        "추가 노트:\n",
+        "\n",
+        "* `tf.contrib`에서 규제를 초기화했다면 다른 것보다 매개변수 변화가 많습니다.\n",
```

Comment:
I don't like to translate '~er' to '~기'.
It is quite used in old literature translated from Japanese.
If necessary, it can be translated '규제 객체' or something more meaningful.
But in this sentence, '규제' is enough to send information. :)